packages/solana/src/utils/chains.ts

```diff
@@ -26,3 +27,9 @@ export const solanaDevnet = {
   rpcUrl: 'https://rpc.walletconnect.org/v1',
   chain: ConstantsUtil.CHAIN.SOLANA
 }
+
+export const solanaChains = {
+  'solana:mainnet': solana,
+  'solana:testnet': solanaTestnet,
+  'solana:devnet': solanaDevnet
+} as Record<`${string}:${string}`, Chain>
```

Comment:
Maybe this?
```suggestion
} as Record<`solana:${"mainnet"|"testnet"|"devnet"}`, Chain>
```

Reply:
This is going to break type assertions forward, maybe we don't need to have as strict?

Reply:
Got it, that's cool then